app/src/main/res/layout/activity_album_review.xml

```diff
@@ -3,6 +3,7 @@
               android:layout_width="match_parent"
               android:layout_height="match_parent"
               xmlns:tools="http://schemas.android.com/tools"
+              xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
xml namespace declarations should be the first elements in the first view in the layout file